docs/includes/config/io.helidon.microprofile.jwt.adoc

```diff
@@ -0,0 +1,82 @@
+///////////////////////////////////////////////////////////////////////////////
```

Comment:
You should use configuration file from `/config` directory and don't use dots in names.